src/lib/components/ng-http-loader.component.html

```diff
@@ -1,4 +1,4 @@
-<div id="spinner" *ngIf="isSpinnerVisible | async">
+<div id="spinner" *ngIf="isSpinnerVisible$ | async">
```

Comment:
Love the Finnish notation! There's a plugin for tslint to enforce this if you're feeling like it.

https://github.com/cartant/rxjs-tslint-rules

```
"rules": {
  "rxjs-finnish": {
    "options": [{
      "functions": false,
      "methods": false,
      "parameters": false,
      "properties": false,
      "variables": true
    }],
    "severity": "error"
  }
}
```

Reply:
That's nice ! I will give this a try.